sdk/generic-sdk-core/generic-core-http-okhttp/src/main/java/com/generic/core/http/okhttp/implementation/OkHttpResponse.java

```diff
@@ -14,9 +14,8 @@
  * Base response class for OkHttp with implementations for response metadata.
  */
 public class OkHttpResponse extends HttpResponse<BinaryData> {
-    private static final BinaryData EMPTY_BODY = BinaryData.fromBytes(new byte[0]);
-
     private final ResponseBody responseBody;
+    private BinaryData body;
```

Comment:
I'm not really understanding the reason why this was removed from being passed to the super class while the behavior seems to be the same as what it was before but now localized to this class

Reply:
I needed to explicitly set the `body` in OkHttpResponse because there's no way to set it in the parent class other than `HttpResponseAccessHelper.setBody()` and I didn't want to add a `setBody()` method to it.

Reply:
Which is why I also had to override the 
`getBody()` method.